src/invoice2data/input/tesseract.py

```diff
@@ -1,29 +1,42 @@
 # -*- coding: utf-8 -*-
 
+from distutils import spawn
+import tempfile
 
-def to_text(path):
-    """Wraps Tesseract OCR.
+from subprocess import Popen, PIPE, STDOUT, CalledProcessError, TimeoutExpired
+from subprocess import run
+from pathlib import Path
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def to_text(path, language="en"):
```

Comment:
This parameter seems to be ignored as below you do:
```
language = get_languages()
```
Can it be just dropped?

Reply:
I left it in for "safety". but if the get_language function is failling, propably tesseract will fail anyway. 
Removing this, as keeping it in could cause confusion.